opendevin/core/config.py

```diff
@@ -173,6 +178,7 @@ class SandboxConfig(metaclass=Singleton):
     use_host_network: bool = False
     initialize_plugins: bool = True
     update_source_code: bool = False
+    od_runtime_extra_deps: str = ''
```

Comment:
Why not also `str | None`? I guess it won't be used too often?

Reply:
fixed!